### PR TITLE
-o flag stdout

### LINKED ISSUE #43: "-o - " flag
Make the following command print to stdout:

```
sb build -o - file.sb
```

### PR DESCRIPTION
**Pull request by garritfra**
Fixes #43

### Description

Adds the option to specify `-o -` as output file to print to stdout.

### ToDo

- [x] Proposed feature/fix is sufficiently tested
- [x] Proposed feature/fix is sufficiently documented
- [x] The "Unreleased" section in the changelog has been updated, if applicable

### CODE REVIEW

**Review by YerinAlexey — APPROVED** [review]

File: src/tests/test_examples.rs [hunk 1]
```diff
@@ -51,7 +51,7 @@ fn test_directory(dir_in: &str) -> Result<(), Error> {
             .spawn()?
             .wait()?
             .success();
-        assert_eq!(success, true, "{:?}", &in_file);
```
Comment (YerinAlexey) [inline src/tests/test_examples.rs:54]: This is out of scope for this patch

Comment (garritfra) [reply src/tests/test_examples.rs]: Superseded by 47ccd2fcb2f2f51ad6f95e3fdc11a2ddbdece8fa

File: - [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/* START builtins */
```
Comment (YerinAlexey) [inline -]: Remove this?
